components/molecules/PullRequestOverview/pull-request-overview.tsx

```diff
@@ -12,58 +11,33 @@ interface PullRequestOverviewProps {
   churnDirection?: string;
   prActiveCount?: number;
 }
+interface OverviewDetails {
+  type: string;
+  percent: number | undefined;
+}
```

Comment:
Refactor (blocking): Narrow the type for `type`. `"open"`, `"closed"`, and `"merged"` seemed to be the only types allowed. Maybe an empty string as well? See my comment below.

```suggestion
interface OverviewDetails {
  type: "open" | "closed" | "merged";
  percent: number | undefined;
}
```